src/goto-instrument/code_contracts.cpp

```diff
@@ -163,6 +163,41 @@ bool code_contractst::has_contract(const irep_idt fun_name)
   return type.has_contract();
 }
 
+void code_contractst::add_quantified_variable(
+  exprt expression,
+  replace_symbolt &replace,
+  irep_idt mode)
+{
+  // If the expression is a quantified expression, this function adds
+  // the quantified variable to the symbol table and to the expression map
+
+  // TODO Currently only works if the contract contains only a single
+  // quantified formula
+  // i.e. (1) the top-level element is a quantifier formula
+  // and (2) there are no inner quantifier formulae
```

Comment:
Can we turn this into a `INVARIANT(...)`? Then we should add a GitHub issue to track that.

Reply:
Could you tag here the GitHub issue to track this comment?

Reply:
I don't think it should be an `INVARIANT` because it can be triggered by user input.  Some kind of warning / error and an issue would be amazing.